pages/windows/taskkill.md

```diff
@@ -0,0 +1,27 @@
+# taskkill
+
+> Terminate a process by its process id or name.
+
+- Display information about the usage of the command:
+
+`taskkill /?`
+
+- Terminate a process by its id:
+
+`taskkill /pid {{process_id}}`
+
+- Terminate a process by its name:
+
+`taskkill /im {{process_name}}`
+
+- Forcefully terminate a specified process:
+
+`taskkill /f`
```

Comment:
How do you specify the process to terminate? Perhaps we could merge this with one of the other examples?

Reply:
I did think about that, but I wasn't sure as it covers the `/pid` and `/im` earlier. I didn't really want to duplicate stuff.

I guess could it mention that it needs to be combined with a process id or name?

Reply:
Yeah, probably. Unless we combine the two examples, I guess?

Reply:
Could I add `(must be combined with a process id or name)` to the end of both descriptions?

Reply:
I think it's ok. It is natural to combine the options from previous examples to have more complex examples and is something the users are expected to do. If you say `/f` and `/pid` and `/im` are mentioned earlier, then that means `/f` can be combined with either of them.

Reply:
Sorry, I think I was not clear. I meant that it was okay as it was before, i.e. without the `(must be combined with a process id or name)`. But this is OK too.